third_party/http_latency_status.t

```diff
@@ -0,0 +1,189 @@
+#!/usr/bin/perl
```

Comment:
could you
1) create a sub-folder "nginx_latency_status_module" for this file
2) add a README.md in that folder to describe the original source of the "t" file
3) use a separate BUILD file for this t test. not to mixed with ngix-tests